test/services/xccdf_report_parser_test.rb

```diff
@@ -184,6 +184,29 @@ class TestParser < ::XccdfReportParser
         )
       end
     end
+
+    should 'raise error if message ID is not present' do
+      assert_raises(::MessageFormatError) do
+        TestParser.new(
+          'fakereport',
+          'account' => accounts(:test).account_number,
+          'b64_identity' => 'b64_fake_identity',
+          'metadata' => { 'fqdn': '123' }
+        )
+      end
+    end
+
+    should 'raise error if metadata fqdn is not present' do
```

Comment:
Why is fqdn required? It seems like we don't use it at all - we update our host's name from the fqdn returned by the inventory request.

Reply:
Before I removed create_host_in_inventory, it was used - now it can be removed and we can validate just `MissingId`